crev-lib/src/local.rs

```diff
@@ -1386,3 +1396,17 @@ fn local_is_send_sync() {
     fn is<T: Send + Sync>() {}
     is::<Local>();
 }
+
+
+fn rm_directory(path_to_delete: &Path) {
```

Comment:
I'm always a bit scared seeing such functions. Get we make add some check here, that the directory is a subdirectory (has a prefix) of the dirs we expect to delete stuff in?